jest.config.ts

```diff
@@ -0,0 +1,194 @@
+/*
+ * For a detailed explanation regarding each configuration property and type check, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+export default {
+  // All imported modules in your tests should be mocked automatically
+  // automock: false,
+
+  // Stop running tests after `n` failures
+  // bail: 0,
+
+  // The directory where Jest should store its cached dependency information
+  // cacheDirectory: "/private/var/folders/fk/c3y07g0576j8_2s9m01pk4qw0000gn/T/jest_dx",
+
+  // Automatically clear mock calls, instances and results before every test
+  clearMocks: true,
```

Comment:
Why has `resetMocks` and `restoreMocks` been replaced by `clearMocks`? `clearMocks` does a subset of what `resetMocks` does, so it seems like the worse choice. We need both `resetMocks` and `restoreMocks` to ensure no mocks or mock state is shared between tests.